prow/config.yaml

```diff
@@ -173,16 +173,6 @@ postsubmits:
         name: jenkins-go
     name: release
 
-  jenkins-x/ext-jacoco:
```

Comment:
Forgot to rename & move this block somewhere.

Reply:
https://github.com/jenkins-x/prow-config/pull/153